NServiceBus scheduler obsolete as of version 8 (#5046)
Particular/docs.particular.net: nservicebus/scheduling/index_content_core_[8,).partial.md

> Executing code on various intervals is a common need when building enterprise systems. NServiceBus provides the following uptions for scheduling:

Reviewer: ```suggestion
Executing code at various intervals is a common need when building enterprise systems. Particular Software proposes the following options for scheduling:
```

Reviewer: I'm not sure we should use the term "NServiceBus provides", because we apart from sagas and the samples, the rest is non-nsb
